manifests/k/Kaseya/LiveConnect/9.5.8299.23994/Kaseya.LiveConnect.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Kaseya.LiveConnect
+PackageVersion: 9.5.8299.23994
+PackageLocale: en-US
+Publisher: Kaseya
+PackageName: Live Connect
+License: Proprietary
+Copyright: Copyright © 2019 Kaseya International Limited
+ShortDescription: Kaseya Live Connect
```

```suggestion
ShortDescription: This stealth-mode remote control tool features an array of gadgets to help maximize service quality and IT efficiency without disturbing end-users.
```
ShortDescription should be meaningful.